lib/marqeta/api_object.rb

```diff
@@ -1,4 +1,5 @@
 require 'marqeta/api_caller'
+require 'time'
```

Comment:
What is this needed for?

Reply:
Oh interesting. I guess we don't need it for the Financeit codebase since Rails probably takes care of it.